release_notes/Service_Fabric_ReleaseNotes_91.md

```diff
@@ -0,0 +1,125 @@
+    ```
+3. Wait for 1-2 mins and policy should get updated across all entities.
+4. Periodic backups will start happening as per backup policy and it can be confirmed by enumerating them.
+
+## Previous Breaking Changes
+
+
+
+## Key Announcements
+* Azure Service Fabric will block deployments that do not meet Silver or Gold durability requirements starting on 10/30/2022. 5 VMs or more will be enforced with this change for newer clusters created after *11/10/2022* to help avoid data loss from VM-level infrastructure requests for production workloads.VM count requirement is not changing for Bronze durability. Enforcement for existing clusters will be rolled out in the coming months. <br> For details see: [Durability characteristics of the cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity#durability-characteristics-of-the-cluster). 
+* Azure Service Fabric node types with VMSS durability of Silver or Gold should always have the property  "virtualMachineProfile.osProfile.windowsConfiguration.enableAutomaticUpdates" set to false in the scale set model definition. Setting this will prevent unintended OS restarts due to the Windows updates like patching, which can impact the production workloads. Instead you should enable Automatic OS upgrades through VMSS OS Image updates by setting "enableAutomaticOSUpgrade" set to true. With automatic OS image upgrades enabled on your scaleset, an extra patching process through Windows Update is not required. <br>
+For more information see: [VMSS Image Upgrades](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade)
+
+## Service Fabric Feature and Bug Fixes
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.1.1390.9590<br>Ubuntu 18 - 9.1.1206.1<br>Ubuntu 20 - 9.1.1206.1** | **Bug** | Cluster Manager service | **Brief Description**: If Upgrade application was called against an application that did not exist then "ApplicationTypeNotFound" is returned instead of "ApplicationNotFound".<br>**Fix**: The code bug in Cluster Manager is fixed to return the correct error message of "ApplicationNotFound"
+| **Windows - 9.1.1390.9590<br>Ubuntu 18 - 9.1.1206.1<br>Ubuntu 20 - 9.1.1206.1** | **Bug** | Managed KeyvaultReferences | **Brief Description**: When deploying an application that uses Managed KeyVaultReferences, customers may previously notice that despite the application starting successfully, the logical “application” resource deployment may fail with AppUpgradeConfigDifferentThanTarget.<br>**Fix**: The code bug was fixed in the backend logic to evaluate app deployment success.<br>Documentation Reference: Read more about Managed KeyVaultReferences at Azure Service Fabric here: [Service Fabric application KeyVaultReferences](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-keyvault-references)
```

Comment:
Sentence tense - instead of "... customers may previously notice..." --> "... customers might have previously noticed ..."

Reply:
instead of "... deployment may fail with AppUpgradeConfigDifferentThanTarget", use "... deployment would fail with AppUpgradeConfigDifferentThanTarget"